commands/runcpp.js

```diff
@@ -35,7 +35,7 @@ module.exports = async (msg) => {
 
   await fs.writeFile('main.cpp', source);
   const compilingMessage = 'Compiling & Running...';
-  const reply = reply[RESPONSE]
+  const reply = msg[RESPONSE]
```

Comment:
unless I'm tripping, `msg[RESPONSE]` isn't set anywhere, another oops

Reply:
should be fixed in [619f260](https://github.com/Anastasia-Romanov/discordjs-eval-bot-new/pull/10/commits/619f260039b4dd25ea3c71fb51204f60baa83d40)